.devcontainer/docker-compose.yml

```diff
@@ -17,7 +17,8 @@ services:
     environment:
       REDIS_URL: "redis://redis:6379/0"
       CIRCUITVERSE_USE_SOLR: "false"
-      DOCKER_ENVIRONMENT: 'true'
+      DOCKER_ENVIRONMENT: "true"
+      REMOTE_DEVELOPMENT_PLATFORM: "true"
```

Comment:
dev containers are used for local development as well right ?

Reply:
For locally, the recommended method was to use the docker environment by `bin/docker_run`. 

But anyone can use the provided configuration to start the dev container in vscode. We haven't tested or documented this. This configuration was specifically added for Github Codespaces

Reply:
@tanmoysrt then Can we us the env DEV_CONTAINER to make this more specific ?

Reply:
@tachyons as there is no pre-existing environment variable like DEV_CONTAINER in the codespaces environment. Added the DEV_CONTAINER variable in docker-compose and updated